modules/host/host.go

```diff
@@ -184,25 +179,6 @@ func (h *Host) Info() modules.HostInfo {
 		info.PotentialProfit = info.PotentialProfit.Add(types.PostTax(h.blockHeight, fc.Payout))
 	}
 
-	// Calculate estimated competition (reported in per GB per month). Price
-	// calculated by taking the average of hosts 8-15.
-	var averagePrice types.Currency
-	hosts := h.hostdb.RandomHosts(15)
-	for i, host := range hosts {
-		if i < 8 {
-			continue
-		}
-		averagePrice = averagePrice.Add(host.Price)
-	}
-	if len(hosts) == 0 {
-		return info
-	}
-	averagePrice = averagePrice.Div(types.NewCurrency64(uint64(len(hosts))))
-	// HACK: 4320 is one month, and 1e9 is a GB. Price is reported as per GB
-	// per month.
-	estimatedCost := averagePrice.Mul(types.NewCurrency64(4320)).Mul(types.NewCurrency64(1e9))
-	info.Competition = estimatedCost
```

Comment:
hmm.

Maybe the host doesn't need access to the competitive layout of the network, and instead can just optimize it's behavior and pricing (people have been asking for automatic pricing tools) based on how much attention it's getting from users.